conf_files/device_info.yaml

```diff
@@ -28,5 +28,8 @@ localhost:
         base: /var/huntsman
         images: /tmp/images
         data: data
-messaging:
-    huntsman_pro_ip: localhost
+control:
+    ip_address: localhost
+    directories:
+        base: /var/huntsman
+        images: images
```

Comment:
Not that we can override the base images directory using the fixtures in conftest should probably have these two `images` directory setting be consistent. I changed to `/tmp/images` for the remote end in my PR so that the unit tests wouldn't write to the 'proper' images directory, but there is a better way of doing that now thanks to your changes to the config server test fixture.

Reply:
@AnthonyHorton So are you suggesting to modify the `images` directory here? If so, I am a little confused because I thought the temp directories were generated by `pytest` and won't have the base/directory format?

Reply:
Suggesting changing the `images` entry back from `/tmp/images` to `images`. I made that change because since the switch to sshfs the remote cameras couldn't use the temp directories generated by the test fixtures, and the only way I could make them use a temporary images directory was to change the default config file to point them to `/tmp`. Now that is no longer necessary thanks to your changes to the config server and the test fixtures we can change the default config file back to being an example of how a real config file would look, i.e. `base: /var/huntsman` and `images: images` for both ends of the distributed system.